src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -452,6 +453,9 @@ class NBDWatchCtx : public librbd::UpdateWatchCtx
 static int open_device(const char* path, bool try_load_module = false)
 {
   int nbd = open(path, O_RDWR);
+  bool loaded_module = false;
+  static bool printed_ignore_info = false;
```

Comment:
@liupan1111 I was proposing to call `open_device()` with `try_load_module = true` only once in `do_map`,  i.e. adding a var `bool try_load_module = true` in `do_map`, before `while` loop and set it to false after `open_device()` call.  Why didn't you like this idea? This looks cleaner and more correct to me (no need to try load module more than one time).